### ISSUE TITLE
can't select fullstop

### ISSUE THREAD

**Issue by DavidGretzschel** [supermemo/SuperMemoAssistant.Plugins.PDF#94]
Can't select fullstops. Might be only in the PDF shown in the video, a copy of "The Mind Illuminated".

https://www.loom.com/share/e257277712d54fd79c57698c12390920

**Comment by DavidGretzschel** [comment]
![image](https://user-images.githubusercontent.com/23729465/78810650-e7f16b00-79c8-11ea-839e-590046476bcd.png)

**Comment by DavidGretzschel** [comment]
seems that the fullstops are extracted, but they don't turn blue in the PDF viewer afterwards nor do they turn grey during Selection.

**Comment by alexis-** [comment]
Have you been able to reproduce the same issue with several PDF documents ?

**Comment by DavidGretzschel** [comment]
Nope, but I haven't been doing much. I posted the PDF in question in the Discord, I believe.
